src/main/java/org/folio/util/ErrorCode.java

```diff
@@ -38,7 +38,9 @@
   ERROR_FIELDS_MAPPING_INVENTORY("error.mapping.fieldsMappingInventory", "An error occurred during fields mapping for inventory record"),
   ERROR_FIELDS_MAPPING_INVENTORY_WITH_REASON("error.mapping.fieldsMappingInventoryWithReason", "An error occurred during fields mapping for inventory record, reason: %s, cause: %s"),
   ERROR_FIELDS_MAPPING_SRS("error.mapping.fieldsMappingSrs", "An error occurred during fields mapping for srs record with id: %s, reason: %s, cause: %s"),
-  ERROR_MESSAGE_PLACEHOLDER("error.messagePlaceholder", "%s");
+  ERROR_MESSAGE_PLACEHOLDER("error.messagePlaceholder", "%s"),
+  MAPPING_PROFILE_MISSING("error.mappingProfile.missing", "Mapping profile is missing during export process"),
```

Comment:
should we be sending this in the response? because it is a little misleading, the validation for mapping profile happens when the API is triggered in the Impl class. If the mapping profile is not passed for some reason in the interactions it's the fault of the module not the user. 
So it will confuse the user because they would have included it in the request

Reply:
Ok, it makes sense to remove this check

Reply:
done